src/expr/src/vector_op/cast.rs

```diff
@@ -49,11 +50,29 @@ const PARSE_ERROR_STR_TO_TIME: &str =
 const PARSE_ERROR_STR_TO_DATE: &str = "Can't cast string to date (expected format is YYYY-MM-DD)";
 const PARSE_ERROR_STR_TO_BYTEA: &str = "Invalid Bytea syntax";
 
+#[inline(always)]
+pub fn i32_to_date(d: i32) -> Result<NaiveDateWrapper> {
+    Ok(NaiveDateWrapper::new(
+        NaiveDate::from_num_days_from_ce_opt(d).unwrap() + Days::new(719_163),
```

Comment:
Need comment for this function. The input integer should represent the number of days since the epoch (`1 January 1970`).